schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT, 
```

Comment:
[OPTIONAL] 
- I think you can make the id column not null and auto incremented and also make it a primary key because the PRIMARY KEY constraint uniquely identifies each record in a table which is very important while making queries.

`id int NOT NULL IDENTITY(1,1)`
`PRIMARY KEY (id)`

Reply:
I will check on that, thank you